datastore/src/main/java/com/ds/datastore/BookStore.java

```diff
@@ -8,24 +8,18 @@
 public class BookStore {
 
     @Id
-    @Column(name = "id", nullable = false)
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    private long id;
-
-    @Column(name = "name", length = 50)
-    private String name;
+    @Column(name = "ServerID")
```

Comment:
as above, the convention would be `server_id` in this case (lowercase, snakecase)